Pipfile

```diff
@@ -4,20 +4,162 @@ verify_ssl = false
 name = "pypi"
 
 [packages]
-e1839a8 = {path = ".",editable = true}
+absl-py = "==0.9.0"
```

Comment:
Can you make the `packages` section clean as before?
As far as I know, the original pattern is better in package development.

Reply:
If there is any testing package required for this, please add it in `setup.py`.
Or, move them to `dev-packages` section, via `pipenv install -d <package>`